pgp/pinentry/getpass.py

```diff
@@ -44,15 +44,15 @@ def get_passphrase(self, prompt=None, error_message=None,
             raise KeyError(cache_id)
 
         if prompt:
-            print(prompt, file=stream)
+            stream.write(prompt + '\n')
```

Comment:
as above, `from __future__ import print_function` would make this look more elegant and forward-thinking.